localstack/services/s3/virtual_host.py

```diff
@@ -42,20 +47,16 @@ def __call__(self, request: Request, **kwargs) -> Response:
         copied_headers["Host"] = forward_to_url.netloc
         copied_headers[S3_VIRTUAL_HOST_FORWARDED_HEADER] = request.headers["host"]
         # do not preserve the Host when forwarding (to avoid an endless loop)
```

Comment:
nit: This comment refers to the Proxy instantiation. I think it should be moved to the init as well.